blackboard/gui/result_pane.py

```diff
@@ -0,0 +1,30 @@
+import gi
+gi.require_version('Gtk', '3.0')
+
+from gi.repository import Gtk, Gio, Gdk  # noqa
+
+
+class result_pane(Gtk.Grid):
+    def __init__(self):
+        Gtk.Grid.__init__(self)
```

Comment:
In python 3.x the better way to call a method from the parent class is using `super()` the syntax is as follow
```python
super().__init__()
```
I know, this is must be fixed in main window too